eth/tracers/api_bor.go

```diff
@@ -56,7 +57,8 @@ func (api *API) traceBlock2(ctx context.Context, block *types.Block, config *Tra
 	if config != nil && config.Reexec != nil {
 		reexec = *config.Reexec
 	}
-	statedb, err := api.backend.StateAtBlock(ctx, parent, reexec, nil, true)
```

Comment:
@ferranbt would also love your thoughts on this change as well, which is due to the latest geth merge. Basically, if setting `false` to prefer disk flag looks good to go or not?